packages/yarnpkg-core/sources/MessageName.ts

```diff
@@ -69,7 +69,9 @@ export enum MessageName {
   FROZEN_ARTIFACT_EXCEPTION = 64,
   TELEMETRY_NOTICE = 65,
   PATCH_HUNK_FAILED = 66,
-  INVALID_CONFIGURATION_VALUE = 67
+  INVALID_CONFIGURATION_VALUE = 67,
+  UNUSED_PACKAGE_EXTENSION = 68,
+  UNNEEDED_PACKAGE_EXTENSION = 69,
```

Comment:
Unused and unneeded are too similar imo (isn't something unused also unneeded?). It's difficult to find something perfect, but what about?

```suggestion
  UNUSED_PACKAGE_EXTENSION = 68,
  SUPERFLOUS_PACKAGE_EXTENSION = 69,
```

Or perhaps the report could explain more:

```
YN0068 Package extension {}: No packages match the selector anymore
YN0069 Package extension {}: No packages seem to benefit from this extension anymore
```

Reply:
> isn't something unused also unneeded?

You're right. 😅 

> `SUPERFLOUS_PACKAGE_EXTENSION = 69,`

Isn't "redundant" more widely used than "superflous" (I admit I had to google it to make sure that it meant what I thought it meant and it's apparently spelled "superfluous")?

If I understand [this StackExchange answer](https://english.stackexchange.com/a/110034) correctly, technically "redundant" would be better here, as a redundant package extension involves an unnecessary repetition, right?

> `YN0069 Package extension {}: No packages seem to benefit from this extension anymore`

Aren't we certain about it so that we can drop the `seem to`?